### DISCUSSION TITLE
Copilot is unavailable!! ---"You don't have access to GitHub Copilot."

### DISCUSSION THREAD

**Discussion by yxw770** [community/community#114546] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I really need help, I don't know what's happened, One day morning, try to use Copilot when I opened my phpstorm ide. But Copilot needed me to log in again and it told me I didn't have access to GitHub Copilot.
![image](https://github.com/community/community/assets/10827451/2d71e5d4-feee-4588-b083-21b67719d147)
![image](https://github.com/community/community/assets/10827451/d9b5666b-ac1e-40ca-9e65-a2a2ac373330)
![image](https://github.com/community/community/assets/10827451/1c244d41-97c9-4df2-a74f-32c72f496c4e)
![image](https://github.com/community/community/assets/10827451/af5f5184-44f2-405c-b91c-26bb419f8913)

**Comment by jrodownz** [comment]
Also running in to this as well, have verified everything on the github admin is good.

**Reply by haileto** [reply]
same problem with me  ， I  have payed it  but not work，it happend yesterday  still now

**Reply by iam24** [reply]
same problem;  china mainland

**Reply by heroyf** [reply]
same problem, today it can no worked in my ide

**Reply by jeffyqwang** [reply]
same problem, I have checked all my settings and tried all the methods mentioned in other issues

**Reply by KMSorSMS** [reply]
same problem, I have checked everything possible but still not working.

**Reply by Adam8en** [reply]
china mainland +1

**Comment by yindehao** [comment]
same problem +1

**Comment by raicdev** [comment]
same problem

**Comment by jarieshan** [comment]
same problem

**Comment by saifeddineelhanoune** [comment]
Double-check if your GitHub account has access to GitHub Copilot You can do this by visiting the GitHub Copilot page (https://copilot.github.com/) and checking if your account is listed as having access

**Reply by jarieshan** [reply]
<img width="1298" alt="image" src="https://github.com/community/community/assets/45466320/0b5cd21c-b819-4ec4-b7e3-771931d00f02">
<img width="947" alt="image" src="https://github.com/community/community/assets/45466320/60b5ee6b-d081-406f-a5fa-6dd4d1ab5a55">
<img width="1009" alt="image" src="https://github.com/community/community/assets/45466320/33d3005a-1a23-4394-aa66-2af6cde67003">

**Reply by yxw770** [reply]
check my image at my first post

**Reply by yxw770** [reply]
we need official help,user can't help us,that problem is copilot reason

**Reply by yxw770** [reply]
I have tried all IDEs and logged in to my copilot on different computers, but all failed. I have applied for assistance but have not received a reply, so I sent for help.

**Comment by hakula139** [comment]
<img width="1136" alt="image" src="https://github.com/community/community/assets/9496568/d4bb3a7f-2cf4-47aa-af2a-14c754a64d1a">
<img width="1101" alt="image" src="https://github.com/community/community/assets/9496568/57a35a4a-e5c5-4d46-8124-3b35b897e239">

+1, China mainland.

**Reply by jeffyqwang** [reply]
me too, china mainland

**Reply by hakula139** [reply]
Will switch to Codeium if not getting responses from the support ticket 🤔

**Comment by SerikaG** [comment]
same problem.

**Reply by SerikaG** [reply]
I've opened a support ticket and they told me that, to quote, "It seems that your access to Copilot has been restricted because our system has detected excessive or automated usage, or proxy usage, which is against our Terms of Service."

I don't know how they judge "excessive or automated usage" when I just use it about 7 hours a day through only one device. But restrict using proxy? My database is on my private server, so do my private repository and other tools I need. I've using proxy switch between different server long before I subscribe copilot. It's just strange when a plugin which is made for developers banned its user because they use proxy.

**Comment by KevenLi8888** [comment]
Same problem here.

**Comment by mhacbq** [comment]
same problem，i need help (

**Comment by yxw770** [comment]
I tried using another github account to purchase copilot, and it worked normally.
![1711615979344](https://github.com/community/community/assets/10827451/a2c6b00f-a970-4cb4-8acc-dad1b3e83602)
![image](https://github.com/community/community/assets/10827451/c4f4aecc-130d-424c-987f-105b4af2c9ab)

**Reply by jeffyqwang** [reply]
I think the reason is Github read an incorrect copilot subscription status, I have paid for copilot but it still tell me to purchase🤔
<img width="955" alt="image" src="https://github.com/community/community/assets/44120031/972175e0-9ab4-49ce-a9eb-460807dce79a">
<img width="965" alt="image" src="https://github.com/community/community/assets/44120031/49661abd-fb07-4289-83f4-0f4196c2fb5e">

**Reply by jeffyqwang** [reply]
My problem has been solved. Just like what I guessed, when my copilot subsciption status display correctly in my settings, I can pass the subscription validate 
All the things I have done is opening a ticket on the support page and waiting. I don't even know whether opening a ticket works or not : )
<img width="875" alt="image" src="https://github.com/community/community/assets/44120031/d20e85f7-918f-4da9-bd67-75f016f54dd4">
<img width="958" alt="image" src="https://github.com/community/community/assets/44120031/af359a3e-3234-4cbd-98f4-c247e79bdfc8">

**Comment by Akash1134** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by jeffyqwang** [reply]
I‘ve opened a ticket on the Support page, hope you can resolve it ASAP.

**Comment by doveppp** [comment]
Please close everything related to CoCopilt(if uesd).

**Reply by jarieshan** [reply]
I have never used any CoCopilot tools before, and I didn't even know what it was before.

**Reply by yxw770** [reply]
What he said is correct. I have used my own server to build an authorization system, which is convenient for switching back and forth between my studio and my home without having to log in again.
I have passed Akash1134
Maintainer
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our Support page. Please open a ticket with them :)
The method mentioned solves my account problem. Users whose account is not used for commercial purposes can fill in support
![image](https://github.com/community/community/assets/10827451/222d6341-7b4a-4c4b-92ef-16282b4f12e8)
![image](https://github.com/community/community/assets/10827451/9f74a01f-d85a-4d3e-aef3-cc5ffa593ff8)

**Comment by haileto** [comment]
same problem  ，please fix it ，github

**Comment by yxw770** [comment]
[> What he said is correct. I have used my own server to build an authorization system, which is convenient for switching back and forth between my studio and my home without having to log in again. I have passed Akash1134 Maintainer Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our Support page. Please open a ticket with them :) The method mentioned solves my account problem. Users whose account is not used for commercial purposes can fill in support ![image](https://private-user-images.githubusercontent.com/10827451/317931696-222d6341-7b4a-4c4b-92ef-16282b4f12e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2Nzc4OTksIm5iZiI6MTcxMTY3NzU5OSwicGF0aCI6Ii8xMDgyNzQ1MS8zMTc5MzE2OTYtMjIyZDYzNDEtN2I0YS00YzRiLTkyZWYtMTYyODJiNGYxMmU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDAxNTk1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmYzE4ZWU3MTViOTRmZTllMzE0MGZlMDg2YWEyNjg0NDg3M2I0NjhjYjlmYTQ0NWU1YWUzYmU2YTM3NjcxYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-xHwqJNRt-ZqQ9UBj5mTG-sQjmw0NS5i8l0hVSYBozM) ![image](https://private-user-images.githubusercontent.com/10827451/317931782-9f74a01f-d85a-4d3e-aef3-cc5ffa593ff8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE2Nzc4OTksIm5iZiI6MTcxMTY3NzU5OSwicGF0aCI6Ii8xMDgyNzQ1MS8zMTc5MzE3ODItOWY3NGEwMWYtZDg1YS00ZDNlLWFlZjMtY2M1ZmZhNTkzZmY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI5VDAxNTk1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFhYmUwMjk5NmY2Mzg1NWI4NzYyYmQ4OGQ0N2I5YmYxMzFlNTMzYTUzMGZiZWQ1YWM5MmIxM2I5MzZlNWU4NmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zl2VxhydXArbtfThXQZfMlnOZlyCIlJxYmDSSLuCGYc)

](https://github.com/orgs/community/discussions/114546#discussioncomment-8947335)

**Comment by yxw770** [comment, marked as answer]
I have successfully fixed the problem. If your account is not a commercial activity to earn copilot fees, then you can use my method to recover the account, and please do not use any third-party API token authorization.
Open [https://support.github.com/contact/account]( https://support.github.com/contact/account)
![image](https://github.com/community/community/assets/10827451/870e8c77-a0d4-4d4b-80c9-eb55f4bb2f2f)
![image](https://github.com/community/community/assets/10827451/7b5ff651-b485-4681-9ea9-ff101a2ed36f)
Just wait for the official solution

**Reply by Sligcm** [reply]
你好，官方回复你用了多久时间？

**Comment by zjrwdl** [comment]
same problem

**Comment by jeffyqwang** [comment]
My problem has been solved. Just like what I guessed, when my copilot subsciption status display correctly in my settings, I can pass the subscription validate in my VScode
All the things I have done is opening a ticket on the support page and waiting. I don't even know whether opening a ticket works or not : )
<img width="958" alt="image" src="https://github.com/community/community/assets/44120031/ca9ac2d3-9b60-4234-9733-3b263c7a1ce7">
<img width="875" alt="image" src="https://github.com/community/community/assets/44120031/80202c68-5c74-49fb-9c31-4318b9bd9167">

**Reply by saifeddineelhanoune** [reply]
great solving

**Comment by airingursb** [comment]
same +1

**Reply by yxw770** [reply]
check this https://github.com/orgs/community/discussions/114546#discussioncomment-8947369

**Reply by airingursb** [reply]
Thanks, it works

**Reply by yxw770** [reply]
加个v嘛？到时候一起交流

**Comment by wpp2014** [comment]


**Reply by Sligcm** [reply]
好的，谢谢，我已经解封了。兄弟，你的姓名微信号和手机号就这么暴露了啊，要懂得保护自己哈。
